packages/web-api/src/config.ts

```diff
@@ -31,6 +32,40 @@ const envSchema = z.object({
     .string()
     .default('false')
     .transform(val => ['true'].includes(val.toLowerCase())),
+  // Comma separated list of job types to not cache
+  DISABLE_SPECIFIC_CACHES: z
+    .string()
+    .optional()
+    .transform((val, ctx) => {
+      // If provided, split and parse into job types
+      if (val) {
+        const validJobTypes: JobType[] = [];
+        for (const item of val.split(',')) {
+          try {
+            const cleanedJobType = item.trim().toUpperCase();
+            const jobType = JobType[cleanedJobType as keyof typeof JobType];
```

Comment:
TypeScript nit, but unless it causes problems, would put that `as ...` type on the `cleanedJobType` const